dev/InfoBar/InfoBarPanel.cpp

```diff
@@ -61,18 +69,18 @@ winrt::Size InfoBarPanel::MeasureOverride(winrt::Size const& availableSize)
     if (nItems == 1 || totalWidth > availableSize.Width || (minHeight > 0 && heightOfTallestInHorizontal > minHeight))
     {
         m_isVertical = true;
-        const auto verticalMargin = PaddingInVerticalOrientation();
+        const auto verticalMargin = VerticalOrientationPadding();
```

Comment:
Minor: confusing for this variable to be named "margin"

Reply:
Changed to verticalPadding/horizontalPadding.